core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-listing-panel/dot-pages-listing-panel.component.scss

```diff
@@ -32,6 +32,19 @@
     .p-datatable .p-datatable-tbody tr td.dot-pages-listing__empty-content {
         text-align: center;
     }
+
+    .p-datatable {
+        .p-datatable-wrapper {
+            height: calc(
+                100vh - 350px
+            ); // To prevent the table from resizing when the results are too low to fill the page
+
+            transition: max-height 400ms cubic-bezier(0.86, 0, 0.07, 1);
```

Comment:
@zJaaal Should we create a mixin to reuse this?

Reply:
not sure, I got this transition from the default `transitionOptions` of `p-panel` https://primeng.org/panel

@fmontes what do you think?

Reply:
We can leave it as it is how we discussed